101-vm-with-data-management-gateway/azuredeploy.json

```diff
@@ -0,0 +1,297 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "existingDataFactoryName": {
+      "type":"string",
+      "metadata": {
+        "description": "your existing data factory name"
+      }
+    },
+    "GatewayName": {
```

Comment:
Not sure I follow - the description says the name must be unique and the resources section creates a new resource with this name - so this doesn't match the description.  The readme also says this is a new resource.  So what is the scenario?